Index: e2fsprogs-1.45.4/configure.ac
===================================================================
--- e2fsprogs-1.45.4.orig/configure.ac
+++ e2fsprogs-1.45.4/configure.ac
@@ -917,6 +917,14 @@ if test $cross_compiling = no; then
 else
   AC_CHECK_PROGS(BUILD_CC, gcc cc)
 fi
+dnl GNU/Hurd does not have sys/mount.h
+case "$host_os" in
+gnu*)
+	MOUNT=""
+	;;
+*)
+	MOUNT="sys/mount.h"
+esac
 AC_CHECK_HEADERS(m4_flatten([
 	dirent.h
 	errno.h
@@ -952,7 +960,7 @@ AC_CHECK_HEADERS(m4_flatten([
 	sys/key.h
 	sys/mkdev.h
 	sys/mman.h
-	sys/mount.h
+	$(MOUNT)
 	sys/prctl.h
 	sys/resource.h
 	sys/select.h
@@ -1320,8 +1328,10 @@ esac
 dnl
 dnl Check the available mount options
 dnl
+if test "x$MOUNT" != "x"; then 
 AX_CHECK_MOUNT_OPT(nosuid)
 AX_CHECK_MOUNT_OPT(nodev)
+fi
 dnl Enable LTO for all packages
 dnl
 AC_ARG_ENABLE([lto],
